lib/datasource/docker/common.spec.ts

```diff
@@ -52,5 +52,16 @@ describe(getName(), () => {
       );
       expect(res).toMatchSnapshot();
     });
+
+    it('supports gitlab project images', () => {
+      const res = dockerCommon.getRegistryRepository(
+        'docker/base',
+        'https://registry.mygitlab.test/jobs'
+      );
+      expect(res).toMatchObject({
+        registry: 'https://registry.mygitlab.test',
+        repository: 'jobs/docker/base',
```

Comment:
I disagree with this. `registry` should match whatever `$CI_DEPENDENCY_PROXY_GROUP_IMAGE_PREFIX` is.

Reply:
I forget why I wrote this